BotProject/Templates/CSharp/Scripts/build_runtime.ps1

```diff
@@ -6,4 +6,12 @@ if ((dotnet --version) -lt 3) {
 # This command need dotnet core more than 3.0
 dotnet user-secrets init
 
-dotnet build
+# Merge all streams into stdout
```

Comment:
at the top of this file, the error check should also be 3.1

Reply:
done